### DISCUSSION TITLE
My website is not available in some countries...

### DISCUSSION THREAD

**Discussion by mehditareghi** [vercel/community#2907] [Bug Report]
### Page to Investigate

techpulse.app

### Steps to Reproduce

I am using vercel hobby account. I created my project and putted proper DNS into my domain. but my website is not available in some countries like Iran. As far as I know it's not about filtering... Please help me with that!

**Comment by smaeda-ks** [comment]
Hi @mehditareghi, I believe this is due to US sanctions:
https://github.com/vercel/vercel/discussions/5891#discussioncomment-840956

**Comment by mehditareghi** [comment]
@smaeda-ks I don't think so. I know some websites that are using Vercel and they are working great in Iran.

**Reply by smaeda-ks** [reply]
@mehditareghi how it's failing? What's the actual error you see on the browser? Please also collect https://gist.github.com/smaeda-ks/5997018d5fe5dce44d047299829288e1 and share that with us.

**Reply by mehditareghi** [reply]
I checked everything and finally find the problem. I removed the redirect to www.my-domain.com and It is okay now. IP is okay but cname connection is not in all regions
